doc/node.1

```diff
@@ -182,26 +182,34 @@ Specify ICU data load path. (overrides \fBNODE_ICU_DATA\fR)
 \',\'\-separated list of core modules that should print debug information.
 
 .TP
-.BR NODE_PATH =\fIpath\fR[:\fI...\fR]
-\':\'\-separated list of directories prefixed to the module search path.
+.BR NODE_DISABLE_COLORS =\fI1\fR
```

Comment:
Are you suggesting not highlighting the value at all? This is how it renders for me:

<img width="371" src="https://cloud.githubusercontent.com/assets/115237/20948204/3d3f3442-bc13-11e6-8e9b-c90420beb220.png">

While it seems more common that man pages don't add arguments in the title, it doesn't seem wrong to me to have them highlighted like this:

<img width="306" src="https://cloud.githubusercontent.com/assets/115237/20948331/dd16750c-bc13-11e6-93be-bfa62575598e.png">